### ISSUE TITLE
Move 'The web accessibility basics' resource to Accessibility category

### ISSUE THREAD

**Issue by vikas-parashar** [webgems/webgems#175]
**Describe the bug**
'The web accessibility basics' resource is in HTML category and should be moved to Accessibility category. 

**To Reproduce**
1. Go to https://webgems.io/html
2. See 'The web accessibility basics' resource in the category.

**Expected behavior**
'The web accessibility basics' should be moved to Accessibility category.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
It's good first issue so beginners can pick it up.

**Comment by lostdesign** [comment]
fixed in #179
